### PR TITLE
[sonic2013] Add more details for test levels and cutscenes

### PR DESCRIPTION
**Pull request by DeaTh-G**

### CODE REVIEW

File: hedgedocs/discoveries/sonic2013/cutscenes/levels.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+│   │   ├── ev430
+│   │   ├── ev440
+│   │   ├── ev520
+│   │   ├── ev530
+│   │   ├── ev620
+│   │   ├── ev642
+│   │   ├── ev645
+│   │   ├── ev710
+│   │   ├── ev720
+│   │   ├── ev750
```
Comment (PTKay) [inline hedgedocs/discoveries/sonic2013/cutscenes/levels.md:75]: This file needs to be added to the mkdocs index file (`mkdocs.yml`)

**Review by PTKay — CHANGES_REQUESTED** [review]
The PR seems fine, but the new file will not be seen if you don't add it to the index

**Comment by DeaTh-G** [comment]
Resolved in 403af33.

**Review by PTKay — APPROVED** [review]
LGTM

**Pull request merged**